evidence/cli/command_controller.go

```diff
@@ -26,41 +25,32 @@ func GetCommands() []components.Command {
 	}
 }
 
-func platformToEvidenceUrls(rtDetails *coreConfig.ServerDetails) {
-	rtDetails.ArtifactoryUrl = utils.AddTrailingSlashIfNeeded(rtDetails.Url) + "artifactory/"
-	rtDetails.EvidenceUrl = utils.AddTrailingSlashIfNeeded(rtDetails.Url) + "evidence/"
-}
+var execFunc = exec
```

Comment:
As the `exec` function is used only from this variable, it can be inlined like:
```go
var execFunc = func(command commands.Command) error {
	return commands.Exec(command)
}
```

Reply:
done